instalint-http-daemon/src/main/java/io/instalint/daemon/ResponseMessage.java

```diff
@@ -1,21 +1,19 @@
 package io.instalint.daemon;
 
 import io.instalint.core.AnalyzerResult;
-import java.io.IOException;
-import java.io.OutputStreamWriter;
-import java.util.List;
-import java.util.Map;
-import java.util.Set;
-import java.util.concurrent.atomic.AtomicInteger;
-import java.util.concurrent.atomic.AtomicReference;
-import javax.servlet.ServletOutputStream;
-import javax.servlet.http.HttpServletResponse;
 import org.sonar.api.batch.fs.TextPointer;
 import org.sonar.api.batch.fs.TextRange;
 import org.sonar.api.utils.text.JsonWriter;
 import org.sonarsource.sonarlint.core.client.api.common.analysis.Highlighting;
 import org.sonarsource.sonarlint.core.client.api.common.analysis.Issue;
 
+import javax.servlet.ServletOutputStream;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.io.OutputStreamWriter;
+import java.util.Map;
+import java.util.Set;
+
```

Comment:
Have you configured your IDE to use SonarSource standards? This kind of reorganization of imports should not happen...

Reply:
I reverted my changes.